docs/spanish/how-to-work-on-guide-articles.md

```diff
@@ -1,51 +1,51 @@
 # Cómo trabajar en artículo de la Guía
 
-Con tu yuda, podemos crear un herramienta de referencia accesible que ayudará a millones de personas que están aprendiendo a programar en los próximos años. 💛
+Con tu ayuda, podemos crear una herramienta de referencia accesible que ayudará a millones de personas que están aprendiendo a programar en los próximos años. 💛
 
 Puedes:
 
-- [Ayudarnos creando y editando artúclos de la Guía](#steps-for-creating-and-editing-guide-articles).
+- [Ayudarnos creando y editando articulos de la Guía](#steps-for-creating-and-editing-guide-articles).
 - [Ayudarnos a revisar Pull Requests para artículos de la guía]()
 
 ## Pasos para crear y editar artículos de la Guía
 
 1. 🍴 [Fork este repositorio](https://github.com/freeCodeCamp/freeCodeCamp#fork-destination-box)
-2. 👀️ Sgiue las normas de controbución expuestas a continuación.
-3. 🔧 Propón cambios asombrosos!
-4. 📖 Lee la [guía de buenas prácticas de estilo](/docs/style-guide-for-guide-articles).
-5. 👉 [Haz una Pull Request](https://github.com/freeCodeCamp/freeCodeCamp/compare)
+2. 👀️ Suigue las normas de contribución expuestas a continuación.
+3. 🔧 Propónga cambios asombrosos!
```

Comment:
Most of this article is written using the second person informal (tú) grammatical person. I would continue using this to be consistent.

So, we should keep ***Propón***